kibana-reports/server/backend/opendistro-notificaiton-plugin.ts

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
np: this file has a type: `notificaiton` vs `notification`

Reply:
good catch